webapp/lib/controller.dart

```diff
@@ -11,13 +11,16 @@ Logger log = new Logger('controller.dart');
 
 final NEEDS_REPLY_METRICS_ROOT_COLLECTION_KEY = 'needs_reply_metrics';
 final SYSTEM_EVENTS_ROOT_COLLECTION_KEY = 'system_events';
+final SYSTEM_METRICS_ROOT_COLLECTION_KEY = 'pipeline_system_metrics';
+final DIR_SIZE_METRICS_ROOT_COLLECTION_KEY = 'dir_size_metrics';
 
 enum UIAction {
   userSignedIn,
   userSignedOut,
   signInButtonClicked,
   signOutButtonClicked,
   needsReplyDataUpdated,
+  systemMetricsDataUpdated,
```

Comment:
should we also have a `dirSizeMetricsDataUpdated` event?

Reply:
Yes. Was avoiding to implement this since we don't have actual charts but gone ahead and implemented it anyway